spec/integration/lhm_spec.rb

```diff
@@ -376,7 +376,7 @@
 
         delete = Thread.new do
           10.times do |n|
-            execute("delete from users where id = '#{ n + 1 }'")
+            execute("delete from users where reference = '#{ n }'")
```

Comment:
The test was going to fail in almost any mysql instance using a big enough value of auto_increment_increment.
The records that were inserted allowed MySQL to choose the primary key (the inserts were done setting the value of `reference`, not of the id).
The deletions were going to try to remove the records with a particular id value, which could or could not be there, depending on auto_increment_increment of mysql.
The test that was always going to be meaningful and successful required to delete 10 of the 50 records inserted above.